### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ZejiaJiang**
# Description
Address https://github.com/Azure/azure-sdk-for-java/issues/31637

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java [hunk 1]
```diff
@@ -1308,8 +1308,8 @@ public void close() {
         }
 
         try {
-            // releated with issue https://github.com/Azure/azure-sdk-for-java/issues/25709. When defining ServiceBusProcessorClient as bean in SpringBoot application and throw error in processMessage(), the application can not be shutdown gracefully using ctrl-c.
-            // The cause is completionLock's acquire stucks. So we add a timeout for acquiring lock here to avoid the stuck.
+            // Related with issue https://github.com/Azure/azure-sdk-for-java/issues/25709. When defining ServiceBusProcessorClient as bean in SpringBoot application and throw error in processMessage(), the application can not be shutdown gracefully using ctrl-c.
+            // The cause is completionLock's acquire stuck. So we add a timeout for acquiring lock here to avoid the stuck.
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java:1312]:
```suggestion
            // The cause is completionLock's acquire method is stuck. So we have added a timeout for acquiring the lock, to work around the issue.
```

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendSessionMessageAsyncSample.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public void run() throws InterruptedException {
         List<ServiceBusMessage> messages = Arrays.asList(
             new ServiceBusMessage(BinaryData.fromBytes("Hello".getBytes(UTF_8))).setSessionId(sessionId),
             new ServiceBusMessage(BinaryData.fromBytes("Bonjour".getBytes(UTF_8))).setSessionId(sessionId),
-            new ServiceBusMessage(BinaryData.fromBytes("Guten tag".getBytes(UTF_8))).setSessionId(sessionId)
+            new ServiceBusMessage(BinaryData.fromBytes("Gluten tag".getBytes(UTF_8))).setSessionId(sessionId)
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendSessionMessageAsyncSample.java:77]:
```suggestion
            
```

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendSessionMessageAsyncSample.java [hunk 3]
```diff
@@ -74,7 +74,7 @@ public void run() throws InterruptedException {
         List<ServiceBusMessage> messages = Arrays.asList(
             new ServiceBusMessage(BinaryData.fromBytes("Hello".getBytes(UTF_8))).setSessionId(sessionId),
             new ServiceBusMessage(BinaryData.fromBytes("Bonjour".getBytes(UTF_8))).setSessionId(sessionId),
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendSessionMessageAsyncSample.java:76]:
```suggestion
            new ServiceBusMessage(BinaryData.fromBytes("Bonjour".getBytes(UTF_8))).setSessionId(sessionId)
```

File: .vscode/cspell.json [hunk 4]
```diff
@@ -177,7 +177,6 @@
     "sdk/schemaregistry/azure-data-schemaregistry-apacheavro/**",
     "sdk/servicebus/build/**",
     "sdk/spring/scripts/**",
-    "sdk/servicebus/azure-messaging-servicebus/**",
```
[the remark this answers is not in the document]
Comment (ZejiaJiang) [reply .vscode/cspell.json:180]: I'll ask in the original issue.

[the remark this answers is not in the document]
Comment (ZejiaJiang) [reply .vscode/cspell.json:179]: refer to other PRs, we should remove.

**Pull request merged**